backend/oasst_backend/task_repository.py

```diff
@@ -129,8 +131,9 @@ def bind_frontend_message_id(self, task_id: UUID, frontend_message_id: str):
         task.ack = True
         # ToDo: check race-condition, transaction
```

Comment:
yay ..my old comment can finally be removed when this gets merged. :-)

Reply:
:) removed it